tests/coverage/closure_macro_async.rs

```diff
@@ -15,7 +15,8 @@ macro_rules! bail {
 
 macro_rules! on_error {
     ($value:expr, $error_message:expr) => {
-        $value.or_else(|e| { // FIXME(85000): no coverage in closure macros
+        $value.or_else(|e| {
+            // FIXME(85000): no coverage in closure macros
```

Comment:
Nit: moving this comment (presumably due to `rustfmt`?) is what caused `closure_macro_async.cov-map` to churn and need re-blessing. Moving it back (and blessing again) would make the diff simpler.

However, now that this PR has been approved and is in the queue, feel free to not worry about it. It's not a big deal either way.

Reply:
Ack (both senses). I tried to revert all the unrelated rustfmt changes, but I missed this one because it was comparatively short. I've fixed this and will push an updated version after running local tests and we'll see what wins.

Reply:
I think I would prefer to keep what's in the PR. Reverting that rustfmt-generated change will just cause whoever is the next person to touch this file to stumble on the same thing again.

Alternatively, I would be open to reverting the rustfmt-generated change from this PR and then receiving a separate PR that applies rustfmt on all the files under `tests/coverage`.

Reply:
I'm happy either way. If this goes through as-is, I won't even bother reverting the change, because in this case the non-standard formatting isn't load-bearing anyway.

There are some other `tests/coverage` tests where non-standard formatting *is* load-bearing (because certain things need to be split across multiple lines); I should probably go through and mark those explicitly with no-format attributes at some point.

Reply:
(Is there a standard way to tell `x.py` to format tests that normally wouldn't be touched by `x fmt`, or are people just running `rustfmt` by hand?)

Reply:
Not sure if this is relevant, but what happened in this PR is that rust-analyzer ran rustfmt for me on the files I edited.

> I think I would prefer to keep what's in the PR.

Ack. Will not push.